docs/source/get_started/access_info/access_info_css.ipynb

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
<p>I think the link for 'this guide' is broken</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>